ble/UUID.h

```diff
@@ -42,7 +97,7 @@ class UUID {
      *         different service or characteristics on the BLE device.
      *
      * @param longUUID
-     *          The 128-bit (16-byte) UUID value, MSB first (big-endian).
+     *          The 128-bit (16-byte) UUID value, LSB first (little-endian).
```

Comment:
since BLE_API is also used in mbed-classic, i'm not sure yotta versioning would shield our users from a sudden switch.

I'm happy to consider alternate constructors (and to mark the existing constructor as deprecated).
how about `UUID().setLongMSB()` and `UUID().setLongLSB()`?

I'd also like to draw in others into the discussion. Perhaps we should discuss this within the team.